ZCRMiOS/Classes/APIResponse.swift

```diff
@@ -111,7 +114,7 @@ public class FileAPIResponse : APIResponse
     init(response: HTTPURLResponse, tempLocalUrl: URL?) throws
     {
         self.tempLocalUrl = tempLocalUrl!
-        try super.init(response: response, responseData: nil, responseJSONRootKey : nil )
+        try super.init(response: response,responseData: nil,responseJSONRootKey:"noRootKey")
```

Comment:
This because the super Class (CommonAPIResponse) is of non - Optional type for the property responseJSONRootKey. It was made non-optional because a non-optional value of DATA is always passed to the APIResponse Class by the Handler classes since DATA is the default JSON Root key for all Handlers unless they over write the value

Reply:
Then lets move this to some constant file and have the dummy value as "file" instead of "noRootKey" so that in future if we have more such types, we can have keys for each type.

Reply:
Okay Anna ! The changes will be reflected in the next commit